picasso/src/main/java/com/squareup/picasso3/AssetRequestHandler.java

```diff
@@ -42,19 +43,15 @@
 
   @Override public boolean canHandleRequest(@NonNull Request data) {
     Uri uri = data.uri;
+    if (uri == null) return false;
```

Comment:
Merge with `return` since it's computing a boolean anyway
```java
return uri != null
    && SCHEME_FILE.equals(uri.getScheme())
    && other thing
    && last thing;
```

Reply:
done